diff --git a/src/AppBundle/Controller/MovieController.php b/src/AppBundle/Controller/MovieController.php
index e2cd7d9..1b3c900 100644
--- a/src/AppBundle/Controller/MovieController.php
+++ b/src/AppBundle/Controller/MovieController.php
@@ -2,12 +2,16 @@
 
 namespace AppBundle\Controller;
 
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
 class MovieController extends Controller
 {
-    public function indexAction($name)
+    /**
+     * @Route("/movies/new")
+     */
+    public function newAction()
     {
-        return $this->render('', array('name' => $name));
+        return $this->render('', array());
     }
 }
