diff --git a/src/AppBundle/Entity/Movie.php b/src/AppBundle/Entity/Movie.php
index 9e06f45..79102b4 100644
--- a/src/AppBundle/Entity/Movie.php
+++ b/src/AppBundle/Entity/Movie.php
@@ -46,4 +46,54 @@ class Movie
     {
         return $this->id;
     }
+
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    public function setTitle($title)
+    {
+        $this->title = $title;
+    }
+
+    public function getSamsCharacterName()
+    {
+        return $this->samsCharacterName;
+    }
+
+    public function setSamsCharacterName($samsCharacterName)
+    {
+        $this->samsCharacterName = $samsCharacterName;
+    }
+
+    public function getRating()
+    {
+        return $this->rating;
+    }
+
+    public function setRating($rating)
+    {
+        $this->rating = $rating;
+    }
+
+    public function getIsMainCharacter()
+    {
+        return $this->isMainCharacter;
+    }
+
+    public function setIsMainCharacter($isMainCharacter)
+    {
+        $this->isMainCharacter = $isMainCharacter;
+    }
+
+    public function getReleasedAt()
+    {
+        return $this->releasedAt;
+    }
+
+    public function setReleasedAt($releasedAt)
+    {
+        $this->releasedAt = $releasedAt;
+    }
 }
