diff --git a/src/AppBundle/Entity/Movie.php b/src/AppBundle/Entity/Movie.php
index 110e0a6..357aea8 100644
--- a/src/AppBundle/Entity/Movie.php
+++ b/src/AppBundle/Entity/Movie.php
@@ -5,7 +5,7 @@ namespace AppBundle\Entity;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @ORM\Entity(repositoryClass="MovieRepository")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\MovieRepository")
  * @ORM\Table(name="movie")
  */
 class Movie
diff --git a/src/AppBundle/Entity/MovieRepository.php b/src/AppBundle/Repository/MovieRepository.php
similarity index 88%
rename from src/AppBundle/Entity/MovieRepository.php
rename to src/AppBundle/Repository/MovieRepository.php
index ae2ed0b..7650eb1 100644
--- a/src/AppBundle/Entity/MovieRepository.php
+++ b/src/AppBundle/Repository/MovieRepository.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace AppBundle\Repository;
 
 use Doctrine\ORM\EntityRepository;
 
