diff --git a/src/AppBundle/Controller/MovieController.php b/src/AppBundle/Controller/MovieController.php
index 3141990..f48b894 100644
--- a/src/AppBundle/Controller/MovieController.php
+++ b/src/AppBundle/Controller/MovieController.php
@@ -22,7 +22,7 @@ class MovieController extends Controller
 
         $form->handleRequest($request);
         if ($form->isValid()) {
-            $em = $this->getDoctrine()->getManager();
+            $em = $this->getEm();
             $em->persist($movie);
             $em->flush();
             // todo do some work, like save ...
@@ -43,7 +43,7 @@ class MovieController extends Controller
      */
     public function listAction()
     {
-        $em = $this->getDoctrine()->getManager();
+        $em = $this->getEm();
         $movies = $em->getRepository('AppBundle:Movie')
             ->findAll();
 
@@ -51,4 +51,13 @@ class MovieController extends Controller
             'movies' => $movies,
         ));
     }
+
+    /**
+     * @return \Doctrine\Common\Persistence\ObjectManager|object
+     */
+    private function getEm()
+    {
+        $em = $this->getDoctrine()->getManager();
+        return $em;
+    }
 }
